amida-tech/greyscale
# Task/inba 491 active project column

#### What's this PR do?
Creates new column for `firstActivated` in `Projects` table and only sets project to isActive when button is toggled.

#### Related JIRA tickets:
[491](https://jira.amida-tech.com/browse/INBA-491)

#### How should this be manually tested?
- Create a project
- go through the flow of adding tasks, users etc
- Check to see if project is marked as Active (It shouldn't)
- Toggle button to make project active (it should work this time)

#### Any background context you want to provide?
Schema changed so you will need to re-run an integration test or something. 
`node_modules/.bin/mocha test/user.integration.js --bail -t 10000`

Meeting all conditions before a project can be marked as Active is a different [ticket](https://jira.amida-tech.com/browse/INBA-517)

#### Screenshots (if appropriate):
